tests/fireflyClient.test.ts

```diff
@@ -1082,6 +1083,178 @@ describe("FireflyClient", () => {
     });
   });
 
+  describe("Sockets", () => {
```

Comment:
Sorry might be a stupid question but I'm struggling to find out the difference between these socket tests and the ones written above :D Could you point out?

Reply:
This was an extra describe, thanks!